### ISSUE TITLE
Feature request: Reverse Audio

### ISSUE THREAD

**Issue by faxinadu** [SamWindell/AudioUtils#13]

**Comment by davidhealey** [comment]
Use SoX `sox input.wav output.wav reverse`

**Comment by mkruselj** [comment]
Correct me if I'm wrong, but doesn't SoX mess with the metadata in the audio file? A metadata-non-destructive way of doing it (which is what Signet does by design) would be preferrable.

**Comment by davidhealey** [comment]
> Correct me if I'm wrong, but doesn't SoX mess with the metadata in the audio file? A metadata-non-destructive way of doing it (which is what Signet does by design) would be preferrable.

Yes you are correct, SoX doesn't preserve the meta data. One solution is to use a bash script that copies the metadata across after the SoX action. But a Signet solution would be preferable.

**Comment by SamWindell** [comment]
Added here https://github.com/SamWindell/Signet/releases/tag/v0.2.0
